cmd/soci-snapshotter-grpc/main.go

```diff
@@ -289,9 +290,10 @@ const (
 	dbMetadataType = "db"
 )
 
-func getMetadataStore(rootDir string, config config.Config) (metadata.Store, error) {
+func getMetadataStore(ctx context.Context, rootDir string, config config.Config) (metadata.Store, error) {
 	switch config.MetadataStore {
 	case "", dbMetadataType:
+		log.G(ctx).Debug("initializing metadata db")
```

Comment:
Maybe let's log `rootDir` as a field. It seems to be the only value that changes here. Everything else is static.